akka-stream-tests/src/test/scala/akka/stream/io/FileSinkSpec.scala

```diff
@@ -129,8 +141,9 @@ class FileSinkSpec extends AkkaSpec(UnboundedMailboxConfig) {
 
   }
 
-  private def targetFile(block: File ⇒ Unit) {
```

Comment:
It's OK, no idea why codacy suddently started commenting on the repo. I removed it completely, sorry for the noise